h3ndrk/hulk
# Audio port

## Introduced Changes

Integrates the whistle detection.
Based on #4 

## ToDo / Known Issues

none

## Ideas for Next Iterations (Not This PR)

none

## How to Test

1. hold your ears
2. blow the whistle
3. observe the nao react to the whistle (currently only possible via integration test with game controller in set; there is no communication yet)
